consumer/ue_context_management.go

```diff
@@ -51,16 +61,29 @@ func UeCmRegistration(ue *amf_context.AmfUe, accessType models.AccessType, initi
 		} else {
 			return nil, openapi.ReportError("server no response")
 		}
+
 	case models.AccessType_NON_3_GPP_ACCESS:
 		registrationData := models.AmfNon3GppAccessRegistration{
 			AmfInstanceId: amfSelf.NfId,
 			Guami:         &amfSelf.ServedGuamiList[0],
 			RatType:       ue.RatType,
 		}
-		ctx, cancel := context.WithTimeout(context.TODO(), 30*time.Second)
+
+		non3gppAccessCtx, cancel := context.WithTimeout(ctx, 30*time.Second)
```

Comment:
Question: In all of the previous changes (other files), you "reuse" the `ctx` variable (e.g., `ctx, cancel := context.WithTimeout(ctx, 30*time.Second)` but here you are using a different name (i.e., `gppAccessCtx`). Any specific reason for this? Why not just naming it `ctx` as in the other files?